pkgs/development/libraries/librsvg/default.nix

```diff
@@ -164,6 +161,10 @@ stdenv.mkDerivation (finalAttrs: {
     # https://gitlab.gnome.org/GNOME/librsvg/-/issues/1150
     substituteInPlace rsvg/meson.build \
       --replace-fail 'requires: library_dependencies_sole,' 'requires: [cairo_dep, gio_dep, glib_dep, pixbuf_dep],'
+
+    # Upstream disables loaders.cache generation when cross-compiling, but unnecessarily so.
+    substituteInPlace gdk-pixbuf-loader/meson.build \
+      --replace-fail 'meson.is_cross_build()' 'false'
```

Comment:
Upstream?

Reply:
maybe. they added these guards pretty explicitly [here](https://gitlab.gnome.org/GNOME/librsvg/-/issues/1073) and then [again](https://gitlab.gnome.org/GNOME/librsvg/-/merge_requests/932/diffs?commit_id=c95b5df57f2a76311b065aff0f245d48f44f90ec) for the MSVC-specific case. maybe i open an issue seeing if we can move this logic into `meson_options.txt` to satisfy everyone.

Reply:
To me this looks very similar to https://gitlab.gnome.org/GNOME/librsvg/-/commit/84f24b1f5767f807f8d0442bbf3f149a0defcf78, where at first they had an overly restrictive check, and then loosened it when somebody pointed that out.

Reply:
i opened a new upstream [PR](https://gitlab.gnome.org/GNOME/librsvg/-/merge_requests/1095) for the gdk-pixbuf-loader & now just `fetchpatch`ing that.

that introspection patch made it super easy: thanks for linking it!